ts/features/itwallet/common/utils/itwCredentialStatusAttestationUtils.ts

```diff
@@ -0,0 +1,62 @@
+import {
+  createCryptoContextFor,
+  Credential
+} from "@pagopa/io-react-native-wallet";
+import { isAfter } from "date-fns";
+import { itwEaaProviderBaseUrl } from "../../../../config";
+import { StoredCredential } from "./itwTypesUtils";
+
+export const getCredentialStatusAttestation = async (
+  credential: string,
+  credentialKeyTag: string
+) => {
+  const credentialCryptoContext = createCryptoContextFor(credentialKeyTag);
+
+  const { issuerConf } = await Credential.Status.evaluateIssuerTrust(
+    itwEaaProviderBaseUrl
```

Comment:
Why don't we use the one stored inside the entity configuration of the credential?

Reply:
You are right: addressed in 4e993d7ee30d371de156c660c2ecd36c53b9f5f2.